### PR TITLE
Add Dutch translations and links

### PR DESCRIPTION
**Pull request by Vistaus**
🙂

### CODE REVIEW

File: data/translations.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
         "pt-br": "Portuguese Brazilian",
         "el": "Greek",
         "it": "Italian",
+        "nl": "Dutch"
```
Comment (Huluti) [inline data/translations.json]: Forgot a ,

File: data/translations.json [hunk 2]
```diff
@@ -130,5 +131,28 @@
...
-   }
+   },
+ "nl": {
+        "en": "Engels",
+        "es": "Spaans",
+        "fr": "Frans",
+        "pt-br": "Portugees (Brazilië)",
+        "el": "Grieks",
+        "it": "Italiaans",
+        "nl": "Nederlands"
```
Comment (Huluti) [inline data/translations.json]: Forgot a ,

**Review by Huluti — COMMENTED** [review]
Thank's! The json seems not valid however. Coud you fix them?

**Comment by Vistaus** [comment]
Oops, you're right. Thanks for catching my mistakes - I fixed them just now! 🙂

**Pull request merged**